frontend/src/components/prompt/PopoverPrompt.vue

```diff
@@ -82,30 +82,21 @@ export default {
   computed: {
     positions() {
       const positions = {}
-      if (this.x === 'left') {
-        positions.left = true
-      } else if (this.x === 'right') {
+      if (this.activatorX === 'left') {
         positions.right = true
+      } else if (this.activatorX === 'right') {
+        positions.left = true
       }
-      if (this.y === 'top') {
-        positions.top = true
-        positions.nudgeBottom = 10
-      } else if (this.y === 'bottom') {
+      if (this.activatorY === 'above') {
         positions.bottom = true
+      } else if (this.activatorY === 'below') {
+        positions.nudgeBottom = 10
+        positions.top = true
       }
```

Comment:
I think I found it more intuitive before. You now added `activator-x` and `activator-y` props. But before, these were not the position of the activator relative to the popover, but the position of the popover relative to the activator. When I use the PopoverPrompt component, I primarily think about where in the UI I put the activator, and then from there, on which side the prompt should pop up once the user clicks the activator. I think that was the way it was implemented before (but I didn't test it out).

If that confuses you, how about renaming the existing `x` and `y` props to `popover-x` and `popover-y`, or removing our custom position props entirely and just expose Vuetify's `left`, `right`, `top` and `bottom` props?

Reply:
I always thought about the origin point and never about the target point. What about `origin-x` and `origin-y`?

I would not want to remove our custom position props as someone could define `left` `right` and `top` at the same time. I need to know in which edge the origin with the activator button is. I don't want to provide more options.

Reply:
I don't understand what "origin" means in this context. In my mental model, an origin can't be shifted, it is usually at (0, 0), and therefore origin-x and origin-y don't make sense to me.
Terms I would understand in this context include "direction", "menu-position", "popover", "activator" (although I don't think new devs will easily understand that), "opener", "button", "trigger", "dropdown", ...

Reply:
I was thinking about the transform origin, which is the point where the popover scales up from. Aligning the popover and button to the same axis makes it more intuitive.

If I understand correctly, for the picture in the issue description, you would define x as left and y as below.
Because, to me, the button and popover are aligned to the right.

@usu what do you think? If you agree with Carlo, I will change the attributes to be defined like this:
```
button="right"
popover="below"
```
(for the picture in the issue description)

Reply:
@carlobeltrame I looked at this again and agree that it was not well worded. I also noticed that Vuetify v-menu has some strange quirk with its left and right prop:
https://codepen.io/manuelmeister/pen/wvRpqLj

When I first implemented this, this confused me, because in CSS if something is aligned right then it is aligned to the right side of container and not "from the right side overflowing towards left" 🤷 

I have settled for 
```html
button-align="left"
popover-position="top"
```

which will result in 
```
╭─────────────────────────────────╮ 
│                                 │
│             popover             │
│                                 │
├──────────┬──────────────────────╯
│  button  │
╰──────────╯
```